include/ofxColour.h

```diff
@@ -0,0 +1,259 @@
+#ifndef _ofxColour_h_
+#define _ofxColour_h_
+
+// Copyright OpenFX and contributors to the OpenFX project.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @file ofxColour.h
+Contains the API for colourspace data exchange.
+*/
+
+/** @brief What style of colour management does the host or plug-in offer?
+
+   - Type - string X 1
+   - Property Set - host descriptor (read only), plugin descriptor (read/write), image effect instance (read only)
+   - Default - kOfxImageEffectPropColourManagementStyleNone
+   - Valid Values - This must be one of
+     - ::kOfxImageEffectPropColourManagementNone - no colour management
+     - ::kOfxImageEffectPropColourManagementBasic - only basic colourspaces from the config header may be used
+     - ::kOfxImageEffectPropColourManagementCore - only core colourspaces from the config header may be used
+     - ::kOfxImageEffectPropColourManagementFull - any colourspace from the config header may be used
+     - ::kOfxImageEffectPropColourManagementOCIO - any OCIO config may be used (implies use of the OCIO library)
+
+Hosts should set this property if they will provide colourspace information 
+to plug-ins.
+Plug-ins should set this property if they can use host-provided colourspace 
+information.
+Collectively, the full, core and basic styles are referred to as native
+colour management. OCIO is used as the reference for the colour management
+API, but is not required to implement the native styles.
+
+The colourspace strings used in the native styles are from ofx-native-1.5.ocio,
+which is based on the OCIO ACES Studio built-in config,
+studio-config-v2.1.0_aces-v1.3_ocio-v2.3, and stored for OFX purposes in
+openfx-native-v1.5_aces-v1.3_ocio-v2.3.h (referred to as the config header).
```

Comment:
Minor: It looks like the name now starts with ofx rather than openfx. And should "ofx-native-1.5.ocio" on line 35 be "the config header"?

Reply:
Fixed in b3caf78 and be3de0e.